tasks/build-monthly-updates-as-json.js

```diff
@@ -10,7 +10,7 @@ const filter = require('gulp-filter');
 
 const path = require('path');
 
-const HELP_CENTER_PRODUCTION_URL = 'https://help.akeneo.com/';
+//const HELP_CENTER_PRODUCTION_URL = 'https://help.akeneo.com/';
```

Comment:
You can just remove it. It can be retrieved very easily with git history if needed.